package.json

```diff
@@ -35,6 +35,7 @@
       "\\.(jpg)$": "identity-obj-proxy"
     },
     "setupFilesAfterEnv": [
+      "./node_modules/@react-native-google-signin/google-signin/jest/build/setup.js",
```

Comment:
Is there a meaningful difference between including this here vs. in the `jest-setup.ts` file?

Reply:
I'm not sure. The google-signin docs says to put it here.

Reply:
We have a jest-setup file but not a `jest.config.ts` file. We could, and probably should, move the config from package.json. That also goes for creating eslint and prettier config files. 

I don't think that is the job of this PR though